src/plugins/intel_npu/tools/common/include/tools_helpers.hpp

```diff
@@ -170,11 +191,11 @@ void reshape(ov::OutputVector inputsInfo, InputsInfo& infoMap, std::shared_ptr<o
             }
         }
         model->reshape(newShapes);
-    } else {
+    } else { // FLAGS_shape is empty
         if (device.find("NPU") != std::string::npos ||
             // FIXME: SIT on CPU also requires to bound dynamic shapes
             device.find("CPU") != std::string::npos || device.find("TEMPLATE") != std::string::npos) {
-            boundDynamicShape(model);
+            modelDynamismCheck(model, false);
```

Comment:
IMHO the old naming suited better, as we do not conduct check only rather we modify layouts with other intrusive actions